src/components/languageSelect/languageSelect.tsx

```diff
@@ -0,0 +1,24 @@
+import { Select } from '@navikt/ds-react';
+import { useTranslation } from 'next-i18next';
+import { useRouter } from 'next/router';
+import React from 'react';
+
+const LanguageSelect = () => {
+  const { t } = useTranslation('common');
+  const router = useRouter();
+  const { pathname, asPath, query, locale } = router;
+
+  const onChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    router.push({ pathname, query }, asPath, { locale: event.currentTarget.value });
+  };
+
+  return (
+    <Select label={t('language-select.label')} hideLabel value={locale} onChange={onChange}>
+      <option value="nb">Bokmål</option>
+      <option value="nn">Nynorsk</option>
+      <option value="en">English</option>
```

Comment:
Se på mulighet for å sette valgbare språk basert på hvilke språk som er tilgjengelig i skjemaet